examples/hh/ilql_hh.py

```diff
@@ -1,11 +1,71 @@
 import os
 
-import yaml
 from datasets import load_dataset
 from ppo_hh import create_reward_fn
 
 import trlx
-from trlx.data.configs import TRLConfig
+from trlx.data.default_configs import (
```

Comment:
Now I see this pattern i wonder if we should just merge all the non-method configs into just TRLConfig since they're always used together